src/hotspot/share/gc/shared/gcVMOperations.hpp

```diff
@@ -118,6 +110,7 @@ class VM_GC_Operation: public VM_GC_Sync_Operation {
   uint           _gc_count_before;         // gc count before acquiring PLL
   uint           _full_gc_count_before;    // full gc count before acquiring PLL
```

Comment:
[pre-existing] "PLL" ?  I think that might be obsolete terminology, referring to the "pending list lock"?  I think should be Heap_lock now.